libs/@blockprotocol/graph/src/codegen/postprocess/prepend-banner-comments.ts

```diff
@@ -0,0 +1,20 @@
+import { mustBeDefined } from "../../shared/util/must-be-defined";
+import { typedKeys } from "../../shared/util/typed-object-iter.js";
+import { PostprocessContext } from "../context/postprocess.js";
+
+const bannerComment = () => `/**
+ * This file was automatically generated – do not edit it.
+ * See the respective URLs for the Block Protocol Types this was generated from.
+ */
```

Comment:
Would it make sense to add the types that the file contains in this comment or any of the others? I appreciate that we may have these URLs in other structures within the generated code, but maybe linking to the types in the docs may be helpful?

Reply:
An alternative to that is raised in a `todo` in the code and in the "Next Steps" of the PR :)

Basically to put the versioned URLs of the types into their doc-comments "Generated from http:..."